content/outcomes/application-development/prioritizing-outcomes.md

```diff
@@ -0,0 +1,170 @@
+
+
+### Structure of a User Story
+
+Write user stories in a consistent manner that provides the team with enough context. 
+
+Common parts of a user story include:
+
+#### Title
+
+The title is a short description that explains the content of the story at a glance and is written from the perspective of a target user. The user might be a real human, a generalized persona, or even another software system.
+
+{{% callout %}}
+Example:
+
+_A Sales rep can download a sales proposal as a PDF_
+{{% /callout %}}
+
+#### Description
+
+The main description of a story should explain which type of user wants the feature, what it allows that user to do, and what outcome or benefit this brings about. An established format is:
+
+AS A `type of user`
+
+I WANT `some goal`
+
+SO THAT `benefit for the end user`
+
+{{% callout %}}
+Example:
+
+**AS A** _sales rep_
+
+**I WANT** _to download a PDF version of a sales proposal_
+
+**SO THAT** _I can email it to a prospective customer_
+{{% /callout %}}
+
+#### Acceptance Criteria
+
+Acceptance criteria list the scenarios the product manager uses to verify that the story is  complete. A common format is the [Behavioral Specification](https://en.wikipedia.org/wiki/Behavior-driven_development#Behavioral_specifications) format, also called the [Gherkin](https://en.wikipedia.org/wiki/Cucumber_(software)#Gherkin_language) language:
+
+GIVEN `situation or context`
+
+WHEN `some action is carried out`
+
+THEN `some consequences`
+
+
+{{% callout %}}
+Example:
+
+**GIVEN** _I visit the proposal summary page_
+
+**WHEN** _I click the button labeled “PDF Download”_
+
+**THEN** _a PDF file of the sales proposal downloads to my computer_
+{{% /callout %}}
+
+Acceptance criteria are important because they **clearly describe when the story is _done_**. Without an explicit statement of “done”, the feature risks [scope creep](https://en.wikipedia.org/wiki/Scope_creep). In the above example, the feature is done when the sales proposal  downloads to the user's computer as a PDF file. 
+
+Would it be handy if the user could email the PDF directly from the product? Sure!
+
+Would it be great if the system would automatically prompt the user to follow up on the sale proposal in one week? Heck yeah!
+
+_But_, those features are _out of scope for this story_. The team should create new stories for those features with their own very explicit and tightly focused acceptance criteria. 
+
+#### Resources
+
+Attach anything else helpful explaining the story including mockups, sketches or user flows.
+
+## Describe the advantages of estimating work in terms of risk and complexity instead of time
+
+Simply put: [people are really bad at estimating based on time](https://en.wikipedia.org/wiki/Planning_fallacy). Yet, most projects, software or otherwise, insist on doing so. We recommend estimating tasks based on risk and complexity rather than time to completion.
```

Comment:
First sentence: Please clarify what exactly it is people are bad at when estimating based on time. For example, people are bad at estimating _____________ based on time.
First sentence: remove "really". It's not necessary.
Second sentence: Please rewrite this sentence a bit. It currently reads, "Yet, most projects, software, or otherwise, insist on doing so. What do projects and software insist on doing? Should it read something like, "Yet, most project **developers**, software **developers** insist on doing so?" 
Second sentence: Consider removing "otherwise or define it to be more clear.

Reply:
Updated